### ISSUE TITLE
[BUG / Feature] `group-hover:scale-` is not supported yet.

### ISSUE THREAD

**Issue by piaro** [francoismassart/eslint-plugin-tailwindcss#76]
Thank you for great plugin!

**Describe the bug**
In my react source like this;
```typescript
 <div
  className={clsx(
    selected ? 'scale-x-100' : 'scale-x-0 group-hover:scale-x-10 group-hover:bg-primary-100',
    'absolute bottom-0 w-full h-0.5 bg-white transition-all origin-center',
  )}
></div>
```
plugin will show the message `warning  Classname 'group-hover:scale-x-10' is not a Tailwind CSS class!  tailwindcss/no-custom-classname`

**To Reproduce**
1. Type `group-hover:scale-x-10` in your code.
2. Run lint
3. You will get the message.

**Expected behavior**
No warning would happen.

**Environment (please complete the following information):**
 - OS: macOS
 - Softwares + version used:
     - VSCode: 1.63.2
     - eslint-plugin-tailwindcss: 3.0.0
     - tailwindcss: 3.0.7

**Comment by francoismassart** [comment]
Hi @piaro,
it looks like the error is coming from the fact that **your own tailwind config is not loaded by the plugin**.

By default the TailwindCSS config offers the following values for scale:
```
    scale: {
      0: '0',
      50: '.5',
      75: '.75',
      90: '.9',
      95: '.95',
      100: '1',
      105: '1.05',
      110: '1.1',
      125: '1.25',
      150: '1.5',
    },
```
And there no color named `primary` by default.

Try the following code 

```
<div
  className={clsx(
    selected ? 'scale-x-100' : 'scale-x-0 group-hover:scale-x-50 group-hover:bg-red-100',
    'absolute bottom-0 w-full h-0.5 bg-white transition-all origin-center',
  )}
></div>
```
and it will work, so I believe the issue you are experiencing is due to a custom config not loaded...
